microxrce_agent.cpp

```diff
@@ -38,8 +38,8 @@ void showHelp()
 #else
     std::cout << "    serial <device_name>" << std::endl;
     std::cout << "    pseudo-serial" << std::endl;
-    std::cout << "    udp <local_port> [<discovery_port>]" << std::endl;
-    std::cout << "    tcp <local_port> [<discovery_port>]" << std::endl;
+    std::cout << "    udp <local_port> [<discovery_port>] [--discovery]" << std::endl;
```

Comment:
Change the order to be:
`--discovery discovery_port`

Reply:
Done